src/rbac/config/db/database.py

```
@@ -0,0 +1,43 @@
+                       pool_recycle=300)
+
+
+def direct_get_conn():
+    conn = None
+    try:
+        conn = engine.connect()
+        return conn
+    except SQLAlchemyError as e:
+        print(e)
+        raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                            detail="요청하신 서비스가 잠시 내부적으로 문제가 발생하였습니다.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)